### PR TITLE
fix: ensure keycard pin error message is visible

### LINKED ISSUE #21645: No warning text regarding PIN attempts during login by Keycard
# Bug Report
https://github.com/status-im/status-mobile/pull/21612

## Problem 
[comment]: # (A problem description)

User doesn't know how many attempts left before his card will be locked.

#### Expected behavior vs Actual behavior

Image

https://www.figma.com/design/YGm3igIOAcwMqUVJWCJ6f1/Keycard?node-id=3-89231&node-type=canvas&t=t0IkcfqJEaXgHC90-0

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by seanstrom**
fixes https://github.com/status-im/status-mobile/issues/21645

### Summary

* This PR attempts to resolve an issue with the keycard PIN error message not being visible after failing to enter the correct PIN. Now the error message should be visible when onboarding with Keycard and sign-in with Keycard.
* This PR also updates the verbiage of the error message to match the designs
* This PR also adds some logic to clear the PIN entry after a PIN error has occurred.

#### Platforms

- Android
- iOS

##### Functional

- Onboarding - Login with Keycard 
- Login - Login with Keycard

### Steps to test

- Open Status mobile app
- Choose to recover profile and login with Keycard
- Scan Keycard
- Enter wrong PIN and scan Keycard again
- Verify error message is visible
- Enter the correct PIN and Scan Keycard
- If successful with recovering the profile, then logout of profile
- Attempt to sign-in to profile with Keycard with wrong PIN
- Verify error message is visible

#### After changes iOS

| Login | Onboard |
|---|---|
| IMG_0010 | IMG_0012 |

#### After changes Android

WIP

status: ready

### CODE REVIEW

File: src/status_im/contexts/keycard/pin/enter/view.cljs [hunk 1]
```diff
@@ -14,5 +14,4 @@
       {:icon-name :i/close
        :on-press  events-helper/navigate-back}]
      [quo/page-top {:title (i18n/label :t/enter-keycard-pin)}]
-     [rn/view {:style {:flex 1}}]
```
Comment (flexsurfer) [inline src/status_im/contexts/keycard/pin/enter/view.cljs:17]: so probably this was a reason somehow?

Comment (flexsurfer) [reply src/status_im/contexts/keycard/pin/enter/view.cljs:17]: and probably as i tested only only on one device, it might be a screen size was the reason

**Review by flexsurfer — APPROVED** [review]

**Review by ilmotta — APPROVED** [review]

**Comment by mariia-skrypnyk** [comment]
Hi @seanstrom !

Thanks for your fix.

Checked both platforms and works fine.
PR can be merged!

**Comment by flexsurfer** [comment]
thank you @seanstrom

File: src/status_im/contexts/keycard/pin/enter/view.cljs (hunk 1 above)
Comment (seanstrom) [reply src/status_im/contexts/keycard/pin/enter/view.cljs:17]: I think the issue can be related to how the flex styling can squish other elements down. I've seen this happen before in the settings screens, and that was also caused by some weird flexbox issue.

**Pull request merged**